frontend/packages/client/src/pages/CommunityCreate.js

```diff
@@ -155,11 +158,14 @@ export default function CommunityCreate() {
   const props = {
     finalLabel: 'Publish',
     onSubmit,
-    isSubmitting: creatingCommunity && !error,
+    isSubmitting: (creatingCommunity || data) && !error,
```

Comment:
added data to avoid this error: 
<img width="911" alt="Screen Shot 2022-08-09 at 19 13 29" src="https://user-images.githubusercontent.com/7526740/183771418-81a4158d-93bc-4182-856d-616d80d41c25.png">